### DISCUSSION TITLE
Can drawing use floating point control？

### DISCUSSION THREAD

**Discussion by Lydanne** [fltk-rs/fltk-rs#1279] [Q&A] [answered]
Ability to use finer control units.

![image](https://user-images.githubusercontent.com/39021696/187019296-f0c88974-3f4d-4d89-a71d-e867c058ecf5.png)

**Comment by MoAlyousef** [comment, marked as answer]
Unfortunately not all drawing functions support floating points. 

The fltk-rs/demos repo shows how to integrate other drawing frameworks (if one needs some advanced drawing) into fltk-rs.

**Reply by Lydanne** [reply]
ok
